typescript-timers/interval.ts

```diff
@@ -0,0 +1,14 @@
+function change(): void {
+  const $countdown = document.querySelector('.countdown-display');
+  if (!$countdown) throw new Error('$countdown query has failed');
+  const number = $countdown.textContent;
```

Comment:
Works, but you shouldn't rely on the DOM for data. It would be better to have a variable in TS keep track of the count